libarchive/archive_read_support_format_rar5.c

```diff
@@ -2510,6 +2511,7 @@ static int parse_tables(struct archive_read* a, struct rar5* rar,
 
 	for(i = 0; i < HUFF_TABLE_SIZE;) {
 		uint16_t num;
+		uint16_t n;
```

Comment:
I'd rather not combine the two different `n` variables into a single variable here.